addons/account/views/account_tax_views.xml

```diff
@@ -203,7 +203,6 @@
             <field name="name">Taxes</field>
             <field name="res_model">account.tax</field>
             <field name="view_mode">tree,kanban,form</field>
-            <field name="domain" eval="False"/> <!-- Force empty -->
```

Comment:
Isn't this sometimes used to reset a domain that was set in a previous version ?

Reply:
@Feyensv indeed but it was done in 2016, so I guess we can assume people have upgraded now 204f9d36054005d38b6fa8659a74ef475164302e